spec/spec_helper.rb

```diff
@@ -62,14 +50,33 @@
   # Allows RSpec to persist some state between runs in order to support
   # the `--only-failures` and `--next-failure` CLI options. We recommend
   # you configure your source control system to ignore this file.
-  config.example_status_persistence_file_path = 'spec/examples.txt'
+  config.example_status_persistence_file_path = 'tmp/spec/examples.txt'
   config.disable_monkey_patching!
-  config.warnings = true
+  config.warnings = false
 
   config.default_formatter = 'doc' if config.files_to_run.one?
   config.profile_examples = 10
 
   config.order = :random
 
   Kernel.srand config.seed
+
+  config.before do
+    TestHelper.configure_grpc
```

Comment:
Do you really need it before every single spec? I don't think so you have to do for the optios_spec I saw. Maybe we ca do a similar approach for a timecop?

Reply:
The point is that when changing the config on the fly in your tests - you don't have to worry about the potential leaking of your actions between tests. Which is why it is here.